website/docs/reference/resource-configs/docs.md

```diff
@@ -70,15 +117,29 @@ snapshots:
   - name: snapshot_name
     docs:
       show: true | false
-
 ```
-
 </File>
 
 </TabItem>
 
 <TabItem value="analyses">
 
+You can use the docs property in YAML files, including the `dbt_project.yml`:
```

Comment:
i'm a little confused why the `dbt_project.yml` is supported for analyses when it only has properties?

Reply:
Correct - you cannot configure multiple `analyses` within `dbt_project.yml`. This isn't for any particularly good reason; it's just never been supported.

They must be configured:
- in-file = `{{ config }}` block in `.sql` file
- in properties yaml file
```yaml
# analyses/properties.yml
analyses:
  - name: my_analysis
    ...
```